### ISSUE TITLE
Predix-seed, grunt serve proxy issue

### ISSUE THREAD

**Issue by AnirbanKundu** [PredixDev/predix-seed#9]
When connecting from local to a micorservice deployed in VPC domain, following error is observed on the console : 

`proxy: error - GET /api/data - getaddrinfo ENOTFOUND unison-web-services-dev.run.aws-usw02-pr.ice.predix.io`

Looks like the grunt-proxy is not able to take into consideration the company proxy it needs to go through while talking to a external URL. (In this case the api I'm trying to connect is `http://unison-web-services-dev.run.aws-usw02-pr.ice.predix.io/api/data`). 

I checked any external api that sits on the other side of the firewall, is not accessible to the grunt-proxy. Looks like, we need to tell grunt to use the company proxy while accessing api that is deployed on VPC

Thanks

**Comment by krisadler-ge** [comment]
This was also helpful to me. My problem was with the view services. The default json-proxy setup kept resulting in not-found errors on the URL, which I could access by browser and curl.

I brought in http-proxy-middleware and https-proxy-agent. I created a proxy agent using process.env.https_proxy || process.env.http_proxy. I then passed this agent in as as a configuration option to the http-proxy-middleware being added. I had to update the connect file to change the path match and target url to use the new format. After these changes, the corporate proxy was used properly, and everything worked.
